### DISCUSSION TITLE
The error in Servo due to the Rust version

### DISCUSSION THREAD

**Discussion by hjshin21** [servo/servo#30672] [Q&A]
Hello, all.

I'm looking for the Servo version that matches Rust 1.64. During my search, I've encountered a persistent error, and I'd like to ask about it. 

![image](https://github.com/servo/servo/assets/104689024/fff08365-22a4-4179-8f4e-879960c454bb)

I'm curious whether this error is due to an incompatibility between the Servo version and Rust 1.64 or if it's a fundamental issue that needs to be resolved.

If the error is related to the Servo version, could you provide information about the Servo version that corresponds to Rust 1.64?

Thank you.

**Comment by mrobinson** [comment]
Servo currently only compiles with the version of Servo that is specified in the toolchain file included with the source.

**Reply by hjshin21** [reply]
I'm currently looking for the version of Servo that matches Rust 1.64!

**Reply by sagudev** [reply]
If I understand correctly you are searching for servo version that works with rust 1.64. I think such servo version does not exist as servo requires specific nightly rust version not stable.

**Reply by hjshin21** [reply]
Thank you! Then... is there a specific Rust version closest to 1.64 that is supported by Servo? If possible, I'm also curious about the Servo version.

**Reply by hjshin21** [reply]
For example, rust version 1.63, 1.65, or 1.66, etc...

**Reply by mrobinson** [reply]
Servo currently depends on nightly-only features, so I'm not sure a stable version of rust will work.
